pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/XPathPanelController.java

```diff
@@ -138,32 +162,71 @@ public void invalidateResults(boolean error) {
     }
 
 
+    public void showExportXPathToRuleWizard() throws IOException {
+        // doesn't work for some reason
```

Comment:
This looks like a TODO - is it something for #714 ?

Reply:
Not really, I forgot to remove it from the previous PR (#711). As it stands, the export menu is not bugged, it's just crap by design. I revamped it some time ago, in a PR that should logically follow this one. I'm still working on making the code more robust